hbase-server/src/main/java/org/apache/hadoop/hbase/replication/regionserver/ReplicationSource.java

```diff
@@ -224,6 +224,10 @@ public void init(Configuration conf, FileSystem fs, ReplicationSourceManager man
     this.abortOnError = this.conf.getBoolean("replication.source.regionserver.abort",
       true);
 
+    int duration = this.conf.getInt(MetricsReplicationSourceRefresherChore.DURATION,
```

Comment:
Are we creating a chore for every replication source ? If we have 2-3 peers, this will create that many chores. Instead is it possible to create just 1 chore and initialize with ReplicationSourceManager which has list of all replicationSources and just iterate over them every time this chore wakes up. Thoughts ?